core/client/fs/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -145,26 +150,27 @@ public FileOutStream createFile(AlluxioURI path, CreateFileOptions options)
     } finally {
       mFileSystemContext.releaseMasterClient(masterClient);
     }
-    OutStreamOptions outStreamOptions = options.toOutStreamOptions();
-    outStreamOptions.setUfsPath(status.getUfsPath());
-    outStreamOptions.setMountId(status.getMountId());
-    outStreamOptions.setAcl(status.getAcl());
+
     try {
+      OutStreamOptions outStreamOptions = new OutStreamOptions(options);
```

Comment:
Because now OutStream is created by a constructor that takes proto options during which we try to create a class instance using class name. 

I wrapped that code in OutStreamOptions with try-catch and throw RuntimeException. Code is out of the try-catch now.